### PR TITLE
Hongyli ocp 26541 loop verif

### PR DESCRIPTION
**Pull request by lihongyan1**
Automated test case OCP-26541
https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-26541
V3 run
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/85496/console
Dependent on PR
https://github.com/openshift/cucushift/pull/7623

### CODE REVIEW

**Comment by lihongyan1** [comment]
@juzhao @pruan-rht 
Please review

**Comment by lihongyan1** [comment]
change code and run v3 job again
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/85776/

File: features/step_definitions/common.rb [hunk 1]
```diff
@@ -128,7 +128,7 @@
   end
 end
 
-When /^I run the :([a-z_]*?) client command in loop `(.+?)` with:$/ do |yaml_key, what, table|
+When /^The output of I run the :([a-z_]*?) client command in loop `(.+?)` should( not)? (contain|match) "(.+)"(?: (\d+) times)? with:$/ do |yaml_key, what, negative, match_type, pattern, times, table|
```
Comment (juzhao) [inline features/step_definitions/common.rb:131]: The output of
->
the output of
is better

File: features/step_definitions/common.rb [hunk 2]
```diff
@@ -115,6 +115,33 @@
     table([[pattern]])
 end
 
+When /^the output of the :([a-z_]*?) client command in loop `(.+?)` should( not)? (contain|match) "(.+)"(?: (\d+) times)? with:$/ do |yaml_key, what, negative, match_type, pattern, times, table|
```
Comment (liangxia) [inline features/step_definitions/common.rb:118]: This is really hard to read.

Comment (pruan-rht) [reply features/step_definitions/common.rb:118]: @lihongyan1 not sure if this make it easier to use w/o documentation.  FYI, we already have a step to loop.  You can use this as a reference: https://github.com/openshift/verification-tests/blob/master/features/test/meta.feature

**Pull request closed without merging**